distributions/io.jamulus.Jamulus.prepare.py

```diff
@@ -0,0 +1,26 @@
+#!/usr/bin/python3
+
+import sys
+import json
+
+
+print('Number of arguments:', len(sys.argv), 'arguments.')
```

Comment:
Debugging output?
Once we merge this to master, all debugging output should be removed

Reply:
true. will be done